google/earthenterprise
# 1640: Update and bundle SWIG

Update and bundle a newer version swig to allow full migration of C++11.

To test (on EL6, and Ubuntu):
- Build portable: You should see swig build in the output at this step.
- Install portable
- Verify that it is working.
- Build and install open.
- Verify that it works.

Also review changes to the documentation and the GitHub portable wiki page.
